### ISSUE TITLE
[Suggestion]: Distorted Sickened mob sounds

### ISSUE THREAD

**Issue by Pandaboi25x** [nonamecrackers2/crackers-wither-storm-mod#1767]
### Description of feature request

title explains everything 

:jess:

**Comment by Aimyx** [comment]
yeah, make em dampened to, cause they kinda are covered in some ws flesh, i
don't think that some of them would talk that well

**Comment by LegendaryGodzilla** [comment]
best sugtp5uhlkng r,k.lcsbgljdiov;zkn

**Comment by Herobrine982950291** [comment]
> best sugtp5uhlkng r,k.lcsbgljdiov;zkn

something wrong with ur keyboard?

**Comment by Aimyx** [comment]
I think he's roleplaying the muffled mob sounds i guess

**Comment by Herobrine982950291** [comment]
did not have to send via email XD but ok

**Comment by Aimyx** [comment]
im looking through my email and im too lazy to open another tab just to
talk about fictional monster's flesh covered mouth

**Comment by Herobrine982950291** [comment]
-_-
